docs/get-started/best-practices/best-practices-for-harvard.md

```diff
@@ -0,0 +1,208 @@
+# Securing Your Public Facing Server
+
+## Overview
+
+This document is aimed to provide you with a few concrete actions you can take
+to significantly enhance the security of your devices. This advice can be
+enabled even if your servers are not public facing. However, we strongly
+recommend implementing these steps if your servers are intended to be accessible
+to the internet at large.
+
+All recommendations and guidance are guided by our policy that has specific
+requirements, the current policy/requirements for servers at NERC can be
+found [here](https://policy.security.harvard.edu/all-servers).
+
+!!! danger "Harvard University Security Policy Information"
+    *Please note that all assets deployed to your NERC project must be compliant
+    with University Security policies. Please familiarize yourself with the
+    Harvard University Information Security Policy and your role in securing
+    data: [https://policy.security.harvard.edu/](https://policy.security.harvard.edu/).
+    If you have any questions about how Security should be implemented in the
+    Cloud, please contact your school security officer: [https://security.harvard.edu/](https://security.harvard.edu/).*
```

Comment:
instead of a bare link maybe do soemthing like "[Havard Security Officer](https://security.harvard.edu/)"